actions/send_email_workflow_status/action.yml

```diff
@@ -128,8 +267,18 @@ runs:
         echo "Email subject: [${email_subject}]"
         echo "Email body:" && cat ${body_file}
         echo ""
-
-        echo "Send email..."
-        echo "TODO IMPLEMENT Actual send email"
-        echo "Send email DONE"
+        # create environment variables from subject to pass to next step
+        echo "SUBJECT=$(echo $email_subject)" >> $GITHUB_ENV
+        echo ""
+        echo ""
       shell: bash
+    - name: Send mail
+      uses: dawidd6/action-send-mail@v3.11.0
+      with:
+       server_address: ${{inputs.builderSmtpHost}}
+       ignore_cert: true
+       secure: false
+       subject: ${{ env.SUBJECT }}
+       to: development@avispl.com
```

Comment:
Ideally I would add a secret for "to" and "from" fields.
Maybe for "from" we could already have a variable, I am not sure